src/controllers/users.controller.ts

```diff
@@ -34,4 +40,15 @@ export class UsersController {
 
     return 'User created';
   }
+
+  @Post('/login-user')
+  async loginUser(@Body() loginUser: LoginUser) {
+    const verifyLogin = await this.authService.validateUser(
+      loginUser.email,
+      loginUser.password,
+    );
+
+    if (verifyLogin) return await this.authService.generateJWT(loginUser);
+    else throw new UnauthorizedException();
```

Comment:
IMO, the single line approach works best if you're only using `if` without the `else` clause.

We can see the thrown exception easier at a glance this way 👍 
```suggestion
    if (verifyLogin) {
      return await this.authService.generateJWT(loginUser);
    }
    else {
      throw new UnauthorizedException();
    }
```